atoma-sui/src/client.rs

```diff
@@ -46,31 +43,35 @@ const UPDATE_NODE_TASK_SUBSCRIPTION_METHOD: &str = "update_node_subscription";
 /// The Atoma's contract method name for submitting a node key rotation attestation
 const ROTATE_NODE_PUBLIC_KEY: &str = "rotate_node_public_key";
 
-/// A client for interacting with the Atoma network using the Sui blockchain.
-///
-/// The `AtomaSuiClient` struct provides methods to perform various operations
-/// in the Atoma network, such as registering nodes, subscribing to models and tasks,
-/// and managing transactions. It maintains a wallet context and optionally stores
-/// a node badge representing the client's node registration status.
-pub struct AtomaSuiClient {
-    /// Configuration settings for the Atoma client, including paths and timeouts.
-    config: AtomaSuiConfig,
-
-    /// The wallet context used for managing blockchain interactions.
-    wallet_ctx: WalletContext,
-
-    /// An optional tuple containing the ObjectID and small ID of the node badge,
-    /// which represents the node's registration in the Atoma network.
+/// Client for interacting with Atoma's Sui blockchain functionality
+pub struct Client {
+    /// Configuration settings for the Atoma client
+    config: SuiConfig,
+    /// The Sui client for blockchain interactions
+    client: WalletContext,
```

Comment:
Let's keep the naming `wallet_ctx`, as it not only contains a `SuiClient` but it wraps around other important functionality like keystores, configs, etc.

Reply:
Good point! 9db8e55